tenant_schemas/tests/test_routes.py

```diff
@@ -1,34 +1,55 @@
 from django.conf import settings
 from django.test.client import RequestFactory
-from tenant_schemas import get_public_schema_name
 
 from tenant_schemas.middleware import TenantMiddleware
-from tenant_schemas.tests.models import Tenant
 from tenant_schemas.tests.testcases import BaseTestCase
+from tenant_schemas.utils import get_public_schema_name, get_tenant_model
 
 
 class RoutesTestCase(BaseTestCase):
     @classmethod
     def setUpClass(cls):
         super(RoutesTestCase, cls).setUpClass()
-        settings.SHARED_APPS = ('tenant_schemas', )
+        settings.SHARED_APPS = ('tenant_schemas',
+                                'customers')
         settings.TENANT_APPS = ('dts_test_app',
                                 'django.contrib.contenttypes',
-                                'django.contrib.auth', )
+                                'django.contrib.auth',)
         settings.INSTALLED_APPS = settings.SHARED_APPS + settings.TENANT_APPS
+        cls.available_apps = settings.INSTALLED_APPS
```

Comment:
Variable not used anywhere else? Please move to a separate PR if parallel migrations does not cause this test to fail.